examples/2d/rotation.rs

```diff
@@ -1,3 +1,5 @@
+//! Demonstrates rotating entities in 2D with quaternions.
```

Comment:
```suggestion
//! Demonstrates rotating entities in 2D using quaternions.
```

Just my opinion, feel free to skip if using “with” is sufficient.